<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COVID_19_Travel_SystemClass</class>
 <widget class="QMainWindow" name="COVID_19_Travel_SystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>860</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>860</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>860</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>主页 - 低风险旅行模拟系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="COVID_19_Travel_System.qrc">
    <normaloff>:/COVID_19_Travel_System/Resources/Main.png</normaloff>:/COVID_19_Travel_System/Resources/Main.png</iconset>
  </property>
  <widget class="QWidget" name="MainWidget">
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/COVID_19_Travel_System/Resources/Title.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="AddPassengerGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>281</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>旅行信息创建</string>
    </property>
    <widget class="QPushButton" name="ButtonSubmit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>提交旅行信息</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxBeginID">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxEndID">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxLimitDay">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelLimitHour">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>时</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInsID">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>旅客编号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTravelID">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelLimitDay">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>天</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxLimitHour">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLimit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>限制旅行时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSrc">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>旅行起点</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDst">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>旅行终点</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTo">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(150, 150, 150);</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxBeginHour">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelBeginHour">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>时</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCustomizeBeginTime">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>自定义开始时间</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxBeginDay">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelBeginDay">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>日</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="QueryGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>281</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>旅行信息查询</string>
    </property>
    <widget class="QLabel" name="labelSelect">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>选择旅客</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTravelID">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelLocal">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>当前位置</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUsed">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>已用时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRest">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>剩余时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUsedDay">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>天</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUsedHour">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>时</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRestDay">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>天</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRestHour">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>时</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUsedDay">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRestDay">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUsedHour">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRestHour">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonViewDetails">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查看详细信息</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserStatus">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="MapViewerGroup">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>741</width>
      <height>711</height>
     </rect>
    </property>
    <property name="title">
     <string>实时地图</string>
    </property>
    <widget class="QLabel" name="labelRunTime">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>45</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>时</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRun">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>45</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>系统已运行</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDay">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>45</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelRunDay">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>45</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>天</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHour">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>45</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonProceed">
     <property name="geometry">
      <rect>
       <x>625</x>
       <y>35</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>▶</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPause">
     <property name="geometry">
      <rect>
       <x>675</x>
       <y>35</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string> ▍▍</string>
     </property>
    </widget>
    <widget class="Map" name="MapViewer" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>700</width>
       <height>600</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>600</height>
      </size>
     </property>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>577</y>
        <width>700</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelTools">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>820</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>实用工具</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonVehicleQuery">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>820</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>航班/列车/汽车信息查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCityDangerQuery">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>820</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>城市风险等级查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSupport">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>820</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>在线支持</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Map</class>
   <extends>QWidget</extends>
   <header>Map.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditTravelID</tabstop>
  <tabstop>comboBoxBeginID</tabstop>
  <tabstop>comboBoxEndID</tabstop>
  <tabstop>checkBoxLimit</tabstop>
  <tabstop>spinBoxLimitDay</tabstop>
  <tabstop>spinBoxLimitHour</tabstop>
  <tabstop>ButtonSubmit</tabstop>
  <tabstop>comboBoxTravelID</tabstop>
  <tabstop>textBrowserStatus</tabstop>
  <tabstop>lineEditUsedDay</tabstop>
  <tabstop>lineEditUsedHour</tabstop>
  <tabstop>lineEditRestDay</tabstop>
  <tabstop>lineEditRestHour</tabstop>
  <tabstop>ButtonViewDetails</tabstop>
  <tabstop>lineEditDay</tabstop>
  <tabstop>lineEditHour</tabstop>
  <tabstop>pushButtonProceed</tabstop>
  <tabstop>pushButtonPause</tabstop>
  <tabstop>pushButtonVehicleQuery</tabstop>
  <tabstop>pushButtonCityDangerQuery</tabstop>
 </tabstops>
 <resources>
  <include location="COVID_19_Travel_System.qrc"/>
 </resources>
 <connections/>
</ui>
